frappe/model/db_query.py

```diff
@@ -97,6 +97,7 @@ def execute(
 		strict=True,
 		pluck=None,
 		ignore_ddl=False,
+		*,
```

Comment:
why?

Reply:
It never really gets called as a positional argument. Doing this allows us to make it keyword-only. We can re-arrange keyword-only arguments - and maybe even deprecate them more easily when required.